docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3010,6 +3018,14 @@ the `TableEntry` entity, which has the following fields:
   implement idle-timeout support for the table, if applicable. See
   [Idle-timeout](#sec-idle-timeout) section for more information.
 
+* `resources`, a `repeated GenericTableEntry` to contain a list of
+  target-specific direct resources. For example, if a target supports special
+  direct resources like x-meter and y-counter, then the p4info will contain
+  GenericTable representations of the same and the list here can contain one
+  member for each of the direct resources. If the generic match fields are set
+  in these entries, it is an error. The IDs will be mentioned in the
+  `direct_resource_ids` in the p4info.
```

Comment:
Suggest `P4Info` instead of `p4info` to be consistent with its capitalization in the rest of the spec.